test/unit/utils/users.test.js

```diff
@@ -0,0 +1,90 @@
+/**
+ * This eslint rule is disabled because of https://github.com/nodesecurity/eslint-plugin-security/issues/21
+ * It gives linting errors in testing the DB data with keys from fixtures
+ */
+/* eslint-disable security/detect-object-injection */
+
+const chai = require('chai')
+const { expect } = chai
+
+const usersUtils = require('../../../utils/users')
+const cleanDb = require('../../utils/cleanDb')
+const addUser = require('../../utils/addUser')
+const userDataArray = require('../../fixtures/user/user')()
+const tasks = require('../../../models/tasks')
+
+/**
+ * Test the utils functions and validate the data returned
+ */
+
+describe('users', function () {
+  let userId, taskId
+  let username, userData
+  let task, taskData
```

Comment:
See `addUser` is being called before every test (beforeEach) and it is returning `userId` which will be unique every single time, so it makes sense to have it declared as `let`, same for `task` and `taskId`

Whereas `username`, `userData`, `taskData` will remain constant throught -- so they can be declared as `const`

Does that make sense? 🤔

Reply:
Yes I got your point👍🏻